### DISCUSSION TITLE
Action failing at "Post Cache target directory"

### DISCUSSION THREAD

**Discussion by D4nte** [community/community#26489] [Actions]
<p>I am getting more and more occurences where the GitHub Action from comit-network/comit-rs repo fails at “Post Cache target Directory”: <a href="https://github.com/comit-network/comit-rs/runs/603537340" rel="nofollow noopener">https://github.com/comit-network/comit-rs/runs/603537340</a></p>
<p>This happens especially when it is a bors-ng commit.</p>
<p>I cannot find any log or error that can explain the issue any more details.</p>
<p>Anyone else encountered the issue and was able to fix it?</p>
<p>Edit: Sometimes it fails at the step just after “Post Checkout Sources” <a href="https://github.com/comit-network/comit-rs/pull/2496/checks?check_run_id=603609140" rel="nofollow noopener">https://github.com/comit-network/comit-rs/pull/2496/checks?check_run_id=603609140</a></p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/d4nte">@d4nte</a> , </p>
<p>Sorry for the bas experience. Your issue is caused by no space left on device issue.  This prevented the job to be finished. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/14542c7be04776c983ef0638517fd267a2dcb5cb.png" data-download-href="/uploads/short-url/2TPTev4xVx35HhbuWvdYTgTKB63.png?dl=1" title="no space left 2.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181098008-51c3ca9d-0e39-4ee3-a7e2-a75c6a10dc50.png" alt="no space left 2.png" data-base62-sha1="2TPTev4xVx35HhbuWvdYTgTKB63" width="645" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/14542c7be04776c983ef0638517fd267a2dcb5cb_2_645x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/14542c7be04776c983ef0638517fd267a2dcb5cb_2_967x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/14542c7be04776c983ef0638517fd267a2dcb5cb.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/14542c7be04776c983ef0638517fd267a2dcb5cb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">no space left 2.png</span><span class="informations">1012×784 43.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>There was an out of space issue of hosted runner : <a href="https://github.com/actions/virtual-environments/issues/709" rel="nofollow noopener">https://github.com/actions/virtual-environments/issues/709</a> </p>
<p>Please keep an eye on it for the latest progress . Thank you for your understanding. </p>
